decimal.md

```diff
@@ -0,0 +1,53 @@
+---
+title: DECIMAL
+toc: true
+---
+
+## Description
+
+The `DECIMAL` [data type](data-types.html) stores exact, fixed-point numbers with no limit on digits to the left and right of the decimal point. This type is used when it is important to preserve exact precision, for example, with monetary data. 
+
+## Synonyms
+
+In CockroachDB, the following are synonyms of `DECIMAL` and are implemented identically:
+
+- `DEC` 
+- `NUMERIC` 
+
+## Precision and Scale
+
+Precision (the maximum count of digits in a whole number, both to the left and right of the decimal point) and scale (the maximum count of digits to the right of the decimal point) are always unlimited for a `DECIMAL` value. If you specify these limits by declaring `DECIMAL(precision, scale)` on a column, they will not be enforced. 
+
+## Format
+
+When declaring a decimal, format it as `DECIMAL '1.2345'`. This casts the value as a string, which preserves the number's exact precision.
+
+Alternately, you can cast a float as a decimal: `CAST(1.2345 AS DECIMAL)`. However, note that precision will be limited to 17 digits in total (both to the left and right of the decimal point). 
```

Comment:
Should we note in the documentation that needing to cast a string to a decimal and the loss of precision when casting from a float are bugs?